### DISCUSSION TITLE
Update the Chia version

### DISCUSSION THREAD

**Discussion by seertenedos** [guydavis/machinaris#662] [Q&A] [answered]
**Describe the request**
I would like to be able to use Chia version 1.3.4 but it seems the chia version is not upgradable in the container and the latest version of the container only seems to have verion 1.3.2. 

**Additional comments**
Could the image maybe have an option to download the latest Chia version on startup or something so that there is a way to always get the latest version if you want it?

**Comment by guydavis** [comment, marked as answer]
Greetings! Chia 1.3.4 had been available in the test and develop images of Machinaris for quite a while now.  Please see: https://github.com/guydavis/machinaris/wiki/Releases

**Comment by seertenedos** [comment]
Thanks. I thought development was not exactly stable so we should stay on stable.  The wiki suggested that image always have the latest version of Chia so that is why i expected it in the stable branch within a few days of it being released not a month or so later (based on a monthly cycle for your stable releases that i did not realise before)

Not a complaint it was just i wanted to explain why i thought it should have been in stable in case some doco or something should be updated to prevent others thinking the same thing
